lock/src/main/java/org/infinispan/lock/impl/ClusteredLockModuleLifecycle.java

```diff
@@ -66,7 +66,10 @@ public void cacheManagerStarted(GlobalComponentRegistry gcr) {
 
    private static Configuration createClusteredLockCacheConfiguration() {
       ConfigurationBuilder builder = new ConfigurationBuilder();
-      builder.clustering().cacheMode(CacheMode.REPL_SYNC)
+      // ISPN-9699: We want the highest consistency, but we can't apply capacity factor 0 to replicated caches
+      // This is a requirement for the Vert.x Cluster Manager, were we want to have all the caches - public and internal
+      // With capacity factor 0, so vert.x nodes are like light members not containing any data
+      builder.clustering().cacheMode(CacheMode.DIST_SYNC).hash().numOwners(Integer.MAX_VALUE)
```

Comment:
Maybe we should introduce the ability to configure CF=0 for members of repl caches which would semantically behave like this but still report as REPL

Reply:
This was discussed on the meeting, and we decided to use a DIST cache on locks with max num of owners because for replicated caches apparently is not that simple ... @danberindei @wburns @pruivo any thoughts ?

Reply:
@tristantarrant the replicated mode uses a simplified version of `ConsistentHashs` and totally ignores the CF and `num-owners`. I guess it can be done but we had to drop the `ReplicatedConsistentHash` and do some tunning probably (or replace it with the `DefaultConsistentHash` with `num-owners == MAX_INTEGER`.